components/DeleteButton.jsx

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export default function DeleteButton({ children, onClick }) {
+  return <StyledDeleteButton onClick={onClick}>{children}</StyledDeleteButton>;
+}
```

Comment:
this "Wrapper" Component for your DeleteButton takes the children and onClick properties and passes them down to the onClick prop and children of the styled component. In other words - its doing nothing at all. If you would call the StyledDeleteButton instead of DeleteButton whereever you use it, nothing would change. 
You can export and use styled components in other places just like regular components, because styled components behave exactly like regular components! 
Please remove this DeleteButton Component, rename StyledDeleteButton to DeleteButton and export that as the default instead.